### PR TITLE
fix flaky tests relying on etcdtest

### PR DESCRIPTION
**Pull request by szuecs**
shortcheck should not rely on etcd tests, ignoring all of them and hack around testing.Main to return on running with shortcheck

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: cmd/eskip/write_test.go [hunk 1]
```diff
@@ -15,16 +15,24 @@
...
 	"testing"
+
+	"github.com/zalando/skipper/etcd/etcdtest"
 )
 
 var testEtcdUrls []*url.URL
 
 func TestMain(m *testing.M) {
+	for _, arg := range os.Args {
+		log.Printf("arg: %s", arg)
```
Comment (aryszka) [inline cmd/eskip/write_test.go]: i guess it's debug

Comment (szuecs) [reply cmd/eskip/write_test.go]: thanks I dropped it

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**